MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -4575,7 +4575,8 @@ static uint32_t mvkGetEntryProperty(io_registry_entry_t entry, CFStringRef prope
 	if (pCreateInfo->pEnabledFeatures) {
 		enableFeatures(&_enabledFeatures.robustBufferAccess,
 					   &pCreateInfo->pEnabledFeatures->robustBufferAccess,
-					   &pdFeats2.features.robustBufferAccess, 55);
+					   &pdFeats2.features.robustBufferAccess, 55,
+                       (char*)"Robust Buffer Access");
```

Comment:
I was about to reply to your bug and express skepticism at the possibility of fixing it given how the code is structured, but then I realized how to do it.

The `enableFeatures()` function takes an *array* of booleans to check. Its callers reinterpret the feature structs as arrays, relying on the compiler laying out the `VkBool32` members end to end with no padding, as in an array. (This among other things means that [the feature Doom 2016 needs](https://github.com/KhronosGroup/MoltenVK/issues/1872) is not necessarily `robustBufferAccess`, though it is somewhere in this struct.) If you want to print the name of the feature(s) which caused the check to fail, you'll need to pass an array of `char *` or `string`. Each member of this array should correspond to a member of the struct being checked.

Reply:
Yes. Ultimately, we'd need a way to stringify the members of each of those structs. 

Unfortunately, C++ doesn't offer any reasonable way to do this, and I expect we'd have to manually type out a static array of c-strings containing the names of all of the struct members, for each struct that we want to submit to `enableFeatures()`.